scripts/linters/js_ts_linter.py

```diff
@@ -59,7 +59,11 @@
 # questions, please talk to @srijanreddy98.
 INJECTABLES_TO_IGNORE = [
     'MockIgnoredService', # This file is required for the js-ts-linter-test.
-    'UpgradedServices' # We don't want this service to be present in the index.
+    # We don't want this service to be present in the index.
+    'UpgradedServices',
+    # This file has location injected in the constructor,
+    # and may cause errors as it alters URL in windows.
```

Comment:
Can you add details on "what type of errors?"?

Reply:
Done